web/js/notification-manager.js

```diff
@@ -17,16 +18,30 @@ function init(mainDOMElement) {
 	mainEl = mainDOMElement;
 	bannerEl = document.getElementById("notification-banner");
 	formEl = bannerEl.querySelector("form");
-	msgEl = bannerEl.querySelector(".msg");
+	messageEl = bannerEl.querySelector(".msg");
 	cancelBtnEl = bannerEl.querySelector("button.cancel-btn");
 	okBtnEl = bannerEl.querySelector("button[type=submit]");
 
 	formEl.addEventListener("submit",onFormSubmit,false);
 	cancelBtnEl.addEventListener("click",() => hide(/*result=*/false),false);
 }
 
-function show(msg,isModal = false,isError = false,showCancel = false,onClose = () => {}) {
-	closeCallback = onClose;
+/**
```

Comment:
I am glad you made this change from the individual params to the options object. I had intended to do so myself! :)

Reply:
Less params the better! Glad you thought it was a good choice.